openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -56,17 +56,22 @@ This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiabl
 
 The following aspects are in scope of this interoperability profile:
 
-* Protocol for issuance of the Verifiable Credentials (can be both remote and in-person) (OID4VCI)
-* Protocol for online presentation of Verifiable Credentials (can be both remote and in-person) (OID4VP)
+* Profile of OID4VCI to issue IETF SD-JWT VCs (can be both remote and in-person), including
+  * Wallet Attestation
+* Profile of OID4VP to present IETF SD-JWT VCs
+* Profile of OID4VP over the W3C Digital Credentials API [@w3c.digital_credentials_api] to present 
+  * IETF SD-JWT VCs 
+  * ISO mdocs
 * Protocol for User Authentication by the Wallet at a Verifier (SIOP v2)
```

Comment:
Should we exclude SIOP for now?

Reply:
Yeah that sounds like a good idea imho, but perhaps a different issue and shouldn't be part of this PR?

Reply:
WG call: Paul to file an issue on this topic.

Reply:
https://github.com/openid/oid4vc-haip/issues/120